pkg/cmd/roachtest/test_runner.go

```diff
@@ -328,7 +328,21 @@ func (r *testRunner) Run(
 		// Don't spin up more workers than necessary.
 		parallelism = n * count
 	}
+	for _, testSpec := range tests {
+		//  TODO(babusrithar): remove this once we see enough data in
+		//  nightly runs. This is a temp logic to test spot VMs.
```

Comment:
FWIW, this comment is largely inaccurate at this point. We already tested spot VMs (and several subsequent PRs went in to make spot VM usage smoother for us and other teams). If anything, the TODO here should be to use spot VMs in a more useful way, especially in combination with test selection.

Reply:
Thanks @renatolabs fro the review! I just copy pasted the old code to here :) - will change this.